ConsoleProject/src/services/TextureConverter.cs

```diff
@@ -182,10 +214,10 @@ public class TextureConverter {
             file.InsertRange(4, BitConverter.GetBytes(1).ToList());
 
             file.RemoveRange(8, 2);
-            file.InsertRange(8, BitConverter.GetBytes(header.FileCount).ToList());
+            file.InsertRange(8, BitConverter.GetBytes((short)header.FileCount).ToList());
```

Comment:
Instead of casting change these fields in TexHeaderMap from integer to short datatype.

Reply:
Thought you changed them and that's what they were originally :D alright, will change.

Reply:
Yes it does ring a bell. I didn't understand the purpose back then.